autopush/metrics.py

```diff
@@ -62,9 +67,12 @@ def timing(self, name, duration, **kwargs):
 class TwistedMetrics(object):
     """Twisted implementation of statsd output"""
     def __init__(self, statsd_host="localhost", statsd_port=8125):
-        self.client = TwistedStatsDClient.create(statsd_host, statsd_port)
+        self.client = TwistedStatsDClient(statsd_host, statsd_port)
         self._metric = Metrics(connection=self.client, namespace="autopush")
 
+    def make_tags(self, base=None, **kwargs):
+        return kwargs
```

Comment:
This throws away the base tags, is that expected?

Reply:
Heh, to be fair, I really should just remove TwistedMetrics since it's no longer active code (aside from unit testing)